addon/styles/_colors.scss

```diff
@@ -1,68 +1,69 @@
 // See https://zeroheight.com/8dd127da7/p/839645-couleurs/b/21317a
+// Before adding a new color here,
+// please check that it is also on ZeroHeight or ask UI Design to add it on ZeroHeight
 
 // primary
 $blue: #3D68FF;
-$yellow: #FFBE00;
 $blue-zodiac: #505F79;
-$black: #07142E;
```

Comment:
C'est un peu dommage de perdre notre $black and $white. D'un point de vue sémantique c'était bien pratique. Par exemple, pour rajouter rapidement et simplement la couleur de police en noir (90% des cas je pense).

Reply:
Est-ce qu'on veut plus coller à Zeroheight ou est-ce qu'on gagnerait pas à avoir quelques différences plus pratique pour développer ?

Reply:
@QuentinChapelain-ui

Reply:
@Anne-Gaelle-S @QuentinChapelain-ui
si on veut garder `$grey-0`, peut-être côté app déclarer `$white: $grey-0`.
(seulement si on tient à garder $grey-0)